.github/workflows/build-unsigned-archive.yml

```
@@ -32,22 +32,30 @@ jobs:
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Install Ruby dependencies
+        run: bundle install
 
       - name: Build Release Version (Unsigned) via Makefile
-        run: make archive
+        run: make build-release
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)